pt_miniscreen/hotspots/templates/status_icon.py

```diff
@@ -1,188 +1,27 @@
 import logging
-from math import ceil, floor
 
-import PIL.Image
-from pitop.miniscreen.oled.assistant import MiniscreenAssistant
+from pt_miniscreen.utils import get_image_file_path
 
 from ...pages.templates.action.state import ActionState
 from ...state import Speeds
-from ..base import Hotspot as HotspotBase
+from .image import Hotspot as ImageHotspot
 
 logger = logging.getLogger(__name__)
 
+image_paths = {
+    ActionState.UNKNOWN: get_image_file_path("status/unknown.png"),
+    ActionState.ENABLED: get_image_file_path("status/enabled.png"),
+    ActionState.DISABLED: get_image_file_path("status/disabled.png"),
+    ActionState.PROCESSING: get_image_file_path("status/processing.gif"),
+    ActionState.IDLE: get_image_file_path("status/idle.png"),
```

Comment:
not sure about having these png/gif replace the drawn-ones ... is this necessary?

Reply:
The drawn ones have three main negatives, the first two related to it cycling through frames:
- It's very difficult to pause / mock out that behaviour in a minimal way to make the tests deterministic.
- it duplicates what is already implemented in the image hotspot in a different way, for rendering gifs.

The last negative is to do with editing the images:
- it's much easier and precise to tweak the images into a pixel editor than to do it in code where we also need to account for a variable size which we never use.

The new implementation allows us to swap out the image that is loaded in the tests to make them consistent and inherit from the image hotspot to remove the duplicate implementation of rendering gifs.